contracts/scripts/core/DeployCore.s.sol

```diff
@@ -15,6 +15,12 @@ import {BlockTracker} from "../../contracts/core/BlockTracker.sol";
 import {console} from "forge-std/console.sol";
 
 contract DeployTestnet is Script {
+
+    // Amount of ETH required to fund the Oracle contract.
```

Comment:
This is being sent the oracle account, not the oracle contract, correct?

Reply:
Yes, fixed.